importAPI/get_job_test.go

```diff
@@ -34,16 +34,46 @@ func TestSuccessfullyGetAnImportJob(t *testing.T) {
 		Convey("When a request to get the job with a specific id and the user is authenticated", func() {
 			Convey("Then the response returns status OK (200)", func() {
 
-				response := importAPI.GET("/jobs/{id}", jobID).WithHeader(tokenName, tokenSecret).Expect().Status(http.StatusOK).JSON().Object()
+				response := importAPI.GET("/jobs/{id}", jobID).
+					WithHeader(serviceAuthTokenName, serviceAuthToken).
+					Expect().Status(http.StatusOK).
+					JSON().Object()
 				checkImportJobResponse(response)
 			})
 		})
+	})
 
-		Convey("When a request to get the job with a specific id and the user is unauthenticated", func() {
-			Convey("Then the response returns status OK (200)", func() {
+	if err := mongo.Teardown(importCreateJobDoc); err != nil {
+		if err != mgo.ErrNotFound {
+			log.ErrorC("Failed to tear down test data", err, nil)
+			os.Exit(1)
+		}
+	}
+}
+
+func TestFailureToGetAnImportJob(t *testing.T) {
+
+	importCreateJobDoc := &mongo.Doc{
+		Database:   cfg.MongoImportsDB,
+		Collection: collection,
+		Key:        "id",
+		Value:      jobID,
+		Update:     validCreatedImportJobData,
+	}
 
-				importAPI.GET("/jobs/{id}", jobID).Expect().Status(http.StatusNotFound)
+	if err := mongo.Setup(importCreateJobDoc); err != nil {
+		log.ErrorC("Failed to set up test data", err, nil)
+		os.Exit(1)
+	}
 
+	importAPI := httpexpect.New(t, cfg.ImportAPIURL)
+
+	Convey("Given an import job exists", t, func() {
+		Convey("When a request to get the job with id does not exist", func() {
```

Comment:
I find this English hard to parse, consider rephrase "... with an id that does not exist" or "... where the id does not exist"?

same for subsequent tests, I'm afraid - perhaps "get the job by ID and..." is appropriate for most, here

Reply:
I have revamped all of the import API tests to hopefully be more clear and concise.